flight/targets/brainre1/board-info/board_hw_defs.c

```diff
@@ -68,7 +71,12 @@ static const struct pios_annunc pios_annuncs[] = {
 		},
 		.remap = 0,
 		.active_high = true,
-	}
+	},
+#if defined(PIOS_INCLUDE_RE1_FPGA)
```

Comment:
True, but I think it's used elsewhere in the same file, so we can also leave it there.